test-kitchen.gemspec

```diff
@@ -30,6 +30,7 @@ Gem::Specification.new do |gem|
   gem.add_dependency "safe_yaml",       "~> 1.0"
   gem.add_dependency "thor",            "~> 0.18"
   gem.add_dependency "mixlib-install",  ">= 1.2", "< 3.0"
+  gem.add_dependency "chef",            "~> 12.17"
```

Comment:
We take pains to avoid a direct dependency on chef keeping test kitchen config manager agnostic. If you absolutely need this, I would recommend making this a separate provisIoner plugin.